src/main/java/core/basesyntax/IFigure.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+interface IFigure {
+    void draw();
+}
```

Comment:
The interface should have a single purpose. If `draw()` is the only method here, then it's fine. However, if you plan to add methods like `getArea()` that are conceptually different, they should be in a separate interface.